src/nonlinear/room-edits.s

```diff
@@ -803,6 +813,14 @@
     .db     0FFh
 .endarea
 
+; Sector 4 - Pump Control Save Room
```

Comment:
is this supposed to be here? if yes, should probably mention it in pr description, IIRC we have an issue for this.

Reply:
I thought I stashed this for a different branch, but I suppose I didn't. We can fix both at the same time. This was originally for #82

Reply:
@AntyMew Would you rather this be in a separate PR? That's how I originally intended it for cleanliness but if you're fine with leaving this in we can merge it as-is.